### PR TITLE
Mat-4145: Update Design System dependencies and validation file

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-4145](https://jira.cms.gov/browse/MAT-4145)
(Optional) Related Tickets:

This PR updates madie-measure a new version of madie-design-system and also removes an unused validation model that currently lives in layout.

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: local-dev-env/hosted-importmap.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "@madie/madie-layout": "//localhost:8500/madie-madie-layout.js",
     "@madie/madie-editor": "//localhost:8501/madie-madie-editor.js",
     "@madie/madie-auth": "//localhost:8502/madie-madie-auth.js",
-    "@madie/madie-cql-library": "//localhost:8507/madie-cql-library.js"
+    "@madie/madie-cql-library": "//localhost:8507/madie-cql-library.js",
+    "@madie/madie-madie-utl": "//localhost:8508/madie-cql-library.js"
```
Comment (adongare) [inline local-dev-env/hosted-importmap.json]: .js seems to be identical. also left side util typo.

File: local-dev-env/importmap.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "@madie/madie-layout": "//localhost:8500/madie-madie-layout.js",
     "@madie/madie-editor": "/madie-editor/madie-madie-editor.js",
     "@madie/madie-auth": "/madie-auth/madie-madie-auth.js",
-    "@madie/maide-cql-library": "/maide-cqllibrary/madie-madie-cql-library.js"
+    "@madie/maide-cql-library": "/maide-cqllibrary/madie-madie-cql-library.js",
+    "@madie/madie-cql-library": "/madie-util/madie-madie-util.js"
```
Comment (adongare) [inline local-dev-env/importmap.json]: the left side is duplicated.

Comment (mcmcphillips) [reply local-dev-env/importmap.json]: thank you

**Review by adongare — APPROVED** [review]
